railties/lib/rails/commands/server/server_command.rb

```diff
@@ -112,12 +112,18 @@ class ServerCommand < Base # :nodoc:
         desc: "Specifies the PID file."
       class_option "dev-caching", aliases: "-C", type: :boolean, default: nil,
         desc: "Specifies whether to perform caching in development."
+      class_option "log-stdout", aliases: "-l", type: :boolean, default: nil,
```

Comment:
Should the alias be `-s` to signify output is going to `STDOUT` or maybe `-v` for verbose? `-l` is good too but I'd like to hear some other opinions on this.

Reply:
I don't think this has enough use to need a short hand.

Reply:
I just removed the shorthand.